src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -62,10 +61,13 @@ public unsafe class WriteAheadJournal : IDisposable
 
         private readonly DisposeOnce<SingleAttempt> _disposeRunner;
 
+        public string EnvironmentName { get; }
```

Comment:
That doesn't actually give us anything, we already report the value back as the path